src/hooks/useMfe.js

```diff
@@ -4,11 +4,8 @@ import { getMfeById } from 'state/mfe/selectors';
 
 
 // try to find the element by id
-const isLoaded = (id) => {
-  const script = document.getElementById(`script-${id}`);
-  const style = document.getElementById(`style-${id}`);
-  return script && style;
-};
+const isJSLoaded = id => document.getElementById(`script-${id}`);
```

Comment:
I wonder maybe we need some kind of `retry with timeout` or not. Is it possible that JS file is not loaded during this check?

Reply:
In this case, we only check if the DOM element of the script is on the page. We didn't have this problem because the script's loading is not related to its DOM injection. The check here is only for preventing from inserting multiple times the same scripts.